cap/01-categorie.tex

```diff
@@ -106,13 +106,33 @@ \section{Categorie}
 \end{example}
 
 In particolare, ogni gruppo si può vedere come una categoria con un solo oggetto. 
+Dato un oggetto $X$ di una categoria, un morfismo da $X$ a $X$ si chiama anche \emph{endomorfismo}. In un monoide, tutti i morfismi sono endomorfismi.
 
 \begin{remark}
  Abbiamo visto che i gruppi sono gli oggetti della categoria $\ctGrp$, ma anche che ogni gruppo si può a sua volta vedere come una categoria (con un solo oggetto). Entrambe le prospettive sono valide, e ce ne sono altre ancora. 
  Allo stesso modo, gli ordini parziali sono categorie, ma si possono anche vedere come oggetti della categoria $\ctPos$, dove i morfismi sono le funzioni monotone. 
  Spesso, in teoria delle categorie, la stessa struttura matematica può apparire in modi diversi in contesti diversi, e questa varietà di prospettive è uno dei motivi per cui le categorie sono così versatili. 
 \end{remark}
 
+Concludiamo questa sezione con alcuni esempi estremi di categorie, utili per controesempi e casi limite.
+
+\begin{examples}
+ Consideriamo un insieme $X$. Possiamo considerarlo una categoria in due modi:
+ \begin{itemize}
+  \item La \emph{categoria discreta su $X$} ha come oggetti gli elementi di $X$, e gli unici morfismi sono le identità.
+  \item La \emph{categoria codiscreta su $X$} ha come oggetti gli elementi di $X$, e un unico morfismo tra ogni due elementi.
+ \end{itemize}
+ La categoria discreta, equivalentemente, è la relazione di equivalenza discreta (la relazione di uguaglianza).
+ La categoria codiscreta si può vedere come un'unica grande classe di equivalenza su $X$.
+\end{examples}
```

Comment:
Qui è chiaro cosa vuoi dire ma ovviamente va spiegato meglio: "la categoria discreta, definita come [bla, \ref{}] coincide con la relazione di equivalenza diagonale come definita in \ref{}, nel senso che l'insieme degli oggetti delle due categorie è chiaramente lo stesso, e hom(x,y) è non vuoto se e solo se... bla bla bla", o qualcosa di simile a questo phrasing.